editor/src/clj/editor/properties_view.clj

```diff
@@ -223,10 +270,15 @@
     (dorun
       (map
         (fn [index ^TextField text-field ^String label-text]
-          (let [children (if (seq label-text)
-                           [(doto (Label. label-text)
-                              (.setMinWidth Region/USE_PREF_SIZE))
-                            text-field]
+          (let [drag-update-fn (fn [v update-val]
+                                 (update v index #(properties/round-scalar (+ % update-val))))
+                children (if (seq label-text)
+                           (let [label (doto (Label. label-text)
+                                         (.setMinWidth Region/USE_PREF_SIZE))]
+                             (ui/do-run-later
+                               #(when-not (properties/read-only? (property-fn))
+                                  (make-label-draggable! label (partial handle-label-drag-event! property-fn drag-update-fn update-ui-fn))))
```

Comment:
We can use `run-later` here since we use an anonymous function. No need to change it here.